conn_test.go

```diff
@@ -374,6 +377,18 @@ func TestMultipleResults(t *testing.T) {
 	if val != 5 {
 		t.Fatalf("expected 5, but found %d", val)
 	}
+
+	rows, err := db.Query("CREATE TEMP TABLE temp (a int); SELECT 6;")
+	if err != nil {
+		t.Fatal(err)
+	}
+	rows.Close()
```

Comment:
Done on the `rows` empty check. I can add a statement that returns a row count, but I can't actually retrieve that count as these tests are going through the `sql` package preventing me from accessing `rows.Result`.